src/main/scala/mesosphere/mesos/TaskBuilder.scala

```diff
@@ -197,6 +196,7 @@ object TaskBuilder {
     // args take precedence over command, if supplied
     app.args.foreach { argv =>
       builder.setShell(false)
+      builder.setValue(argv.head)
```

Comment:
If this was tested already and it somehow worked, I'd be interested in that too.  At the very least, we should verify that both the Mesos containerizer + command executor and the Docker containerizer both still work with both `cmd` and `args`.